testsuite/openshift/objects/auth_config/__init__.py

```diff
@@ -25,7 +25,7 @@ def authorization(self) -> AuthorizationSection:
     @cached_property
     def identity(self) -> IdentitySection:
         """Gives access to identity settings"""
-        return IdentitySection(self, "identity")
+        return IdentitySection(self, "authentication")
```

Comment:
should we also rename `identity` tests package to `authentication` to avoid confusion?

Reply:
I agree, but can be done later